src/transports/tcp/TcpConnectionAcceptor.h

```diff
@@ -53,6 +53,11 @@ class TcpConnectionAcceptor : public ConnectionAcceptor {
    */
   void stop() override;
 
+  /**
+   * Get the port being listened on.
+   */
+  folly::Optional<uint16_t> listeningPort() const override;
```

Comment:
Does this need to be optional at all? Just return value of the options.port should be ok?